#
#
robot:
  - [3.5, 0]
  - [2.5, -1.0]
  - [-1.5, -1.0]
  - [-1.5, 1.0]
  - [2.5, 1.0]
theta_resolution_deg: 0.1
allow_unknown: true
occupancy_threshold: 0
